openaerostruct/functionals/breguet_range.py

```diff
@@ -36,18 +36,19 @@ def setup(self):
             name = surface['name']
             self.add_input(name + '_structural_weight', val=1., units='N')
 
-        self.add_input('CT', val=1., units='1/s')
-        self.add_input('CL', val=1.)
-        self.add_input('CD', val=1.)
-        self.add_input('a', val=1., units='m/s')
-        self.add_input('R', val=1., units='m')
-        self.add_input('M', val=1.)
-        self.add_input('W0', val=1., units='kg')
-        self.add_input('load_factor', val=1.)
+        self.add_input('CT', val=0.25, units='1/s')
+        self.add_input('CL', val=0.7)
+        self.add_input('CD', val=0.02)
+        self.add_input('a', val=100., units='m/s')
+        self.add_input('R', val=3000., units='m')
+        self.add_input('M', val=1.2)
+        self.add_input('W0', val=200., units='kg')
+        self.add_input('load_factor', val=1.05)
 
         self.add_output('fuelburn', val=1., units='kg')
 
         self.declare_partials('*', '*')
+        self.set_check_partial_options(wrt='*', method='fd', form='central')
```

Comment:
Can you get rid of this line so we're not checking against fd? Or did you need that instead of cs?